### ISSUE TITLE
Dinosaurs all stuck in place. Tyrannosaurus still roars, but that's it. No guests show up now either. 

### ISSUE THREAD

**Issue by SpoopyDoopy0704** [PrehistoricKingdom/feedback#277]
Different thing, should probably open up a new thread for it, but my game crashes whenever it autosaves, I manually save, or try to exit to menu.

**Comment by Xul666** [comment]
Got the same issue since yesterday evening!
All my 4 saved parks no longer work.
I can load the parks, but all 

dinos stuck on place, lay down and roar. But they don't move. 
No guests show up - they all stuck in the spawn point under the entrance buildung.
When I click on "return to menu" the game crashes.

This does not only concern one specific save game, but all my parks!
Yesterday afternoon they all run fine, but in the evening the issues came up.

It seems the parks are not initialized correctly (I'm not sure if this is correct description in english - I'm from Germany). It's waiting for a "go" but this is missing...

I made a 3 minute video to show the full situation. 
here is the link: 
https://youtu.be/avWjK5EmJw8

**Comment by PrehistoricKingdom** [comment]
Please try renaming navmesh.data in your save's folder to nav2.data and then load that save again. This will force the game to regenerate the navmesh on load (may cause the loading to take up to a minute). Please let me know if this fixes your issue or not.

**Comment by Xul666** [comment]
No, this did not fixes the issues.
I have renamed the file as described. But this had no effect.
 
I have also noticed that two of my save games have no "navmesh.data" in the folder! 
The autosave not either.
Is this normal?

Then I tried something different:
I loaded the save file. I erased one part of the path and reconnect it. I saved the park under a new name. The game crashed  after saving but the new save folder was created (also without that "navmesh.data").
Then I loaded the new park - and it worked! Guest appered and all dinos were moving instantly.
But unfortunately that only worked once. After ending the game, restarting it and loading the new park, the same issues occur...

**Comment by Xul666** [comment]
I noticed the following:
all saved Parks run fine, when there is no "navmesh.data" in the folder!
But only when the game is started for the first time. When I try to save those parks or load another save or end the game, it crashes.
When I close the game in the task manager and restart it, all saved parks have the issue with missing guests and dinos are stuck.

When I restart my computer, and only then, everything works fine with these saves.
Seems there is something wrong with the "game cache" or any log file or something...?

All saves with a "navmesh.data" never work for me!
But when I delete the file in the save folder I can start them and guests appear and dinos move. When I save those parks, the game crashes but the "navmesh" is re-created.

Now it's your turn again :-)

I'm confused I had no issues on Friday (when the game was released) or an Saturday. But on Sunday evening the issues began. 
Any patch on Sunday I did not notice?

**Comment by Xul666** [comment]
Now ALL save file don't work anymore!
I don't understand how this can happen...
I have already uninstalled and reinstalled the game - no chance to get rid off the issues.

PC specs: 32 GB DDR4 RAM, CPU: Intel(R) Core(TM) i5-7600K CPU @ 3.80GHz
Windows10

Player log:
[Player.log](https://github.com/PrehistoricKingdom/feedback/files/6192287/Player.log)

I also attached one of the "frozen" parks.

[alpha test.zip](https://github.com/PrehistoricKingdom/feedback/files/6192309/alpha.test.zip)
